xml/hardware_nvdimm.xml

```diff
@@ -38,33 +39,29 @@
   <para>
    Like SSDs, NVDIMMs provide non-volatile storage: their contents are retained
    when the system is powered off or restarted. Also like SSDs, sector-level
-   access is also possible if that is more suitable for a particular
-   application.
+   access is possible if that is more suitable for a particular application.
   </para>
 
   <para>
    This new storage subsystem is defined in version 6 of the ACPI standard.
-   However, some types of NVDIMMs already existed before the standard was
-   defined. These can be used in the same way.
-  </para>
-  
-  <para>
-   Different models use different forms of electronic storage medium, such as
-   Intel 3D XPoint, or a combination of NAND flash and DRAM, and in future,
-   memristor storage or other forthcoming types of non-volatile memory.
+   However, NVDIMMs already existed before the standard was defined, meaning
+   that non-compliant hardware exists. These can be used in the same way.
   </para>
 
   <para>
-   This means that different vendors' NVDIMMs offer different performance and
-   durability characteristics. Typically, they are slower than DRAM, but
-   considerably faster than SSDs based on NAND-flash technology.
+   Different models use different forms of electronic storage medium, such as
+   Intel 3D XPoint, or a combination of NAND flash and DRAM. New forms of
+   non-volatile RAM are also in development. This means that different vendors'
+   NVDIMMs offer different performance and durability characteristics.
+   Typically, they are slower than DRAM, but considerably faster than SSDs based
+   on NAND-flash technology.
   </para>
-
+  
   <para>
    Because the storage technologies involved are in an early stage of
    development, different vendors' hardware may impose different limitations.
-   However, generally, writing data to NVDIMMs is slower than reading from
-   them, and the number of write cycles is generally limited.
+   However, generally, writing data to persistent memory is slower than reading
```

Comment:
Do you need both modifications here ('However' and 'generally')? How about the following: 'Generally, writing data to persistent memory is slower than reading from it, and the number of write cycles is limited.'

Reply:
See comment above. I would value your comments on the new paragraph about performance characteristics.